config/crd/bases/operator.kairos.io_nodeops.yaml

```diff
@@ -148,6 +161,11 @@ spec:
                         Phase represents the current phase of the operation on this node.
                         Can be "Pending", "Running", "Completed", or "Failed"
                       type: string
+                    rebootStatus:
+                      description: |-
+                        RebootStatus represents the reboot state of this node.
+                        Can be "false" (reboot not requested), "pending" (reboot requested but not completed), or "completed" (reboot finished)
```

Comment:
no failed status? Becuase the upgrade could work but then the reboot fail for some reason? Or maybe it cant?